app/mailers/group_mailer.rb

```diff
@@ -1,8 +1,26 @@
-class GroupMailer < ApplicationMailer
-	default from: 'no-reply@chalmers.it'
+require 'faraday'
+
+class GroupMailer
```

Comment:
Maybe rename to `GotifyMailer` and move outside of the `mailers` folder to indicate that this is not actually an `ActionMailer` class